### PR TITLE
Add persistence guidelines

### PR DESCRIPTION
**Pull request by andresbono**

### CODE REVIEW

File: stacksmith/user-scripts/boot.sh [hunk 1]
```diff
@@ -33,18 +37,24 @@ installWordPress() {
         --admin_email="email@domain.com"
 }
 
+installPlugins() {
```
Comment (carrodher) [inline stacksmith/user-scripts/boot.sh]: Do we have a way for the user to choose the plugins to install or should he modify this list?
I mean, for sure the user should be able to install plugins using the UI, but at this level, should he be able to do it? If yes, should we document it?

Comment (andresbono) [reply stacksmith/user-scripts/boot.sh]: I have added the feature to use the plugins (zip files) uploaded in the `UPLOADS_DIR` directory. I will document it.

File: stacksmith/user-scripts/build.sh [hunk 2]
```diff
@@ -10,21 +10,30 @@ installDependencies() {
...
     wp --path=$installdir core download
 }
 
+prepareDataToPersist() {
+    chown -R apache "${installdir}/wp-content"
+    mv "${installdir}/wp-content" /opt/
+}
+
 main() {
+    # The directory where WordPress is installed
```
[the remark this answers is not in the document]
Comment (andresbono) [reply stacksmith/user-scripts/build.sh]: Why the copy to `/opt/plugins`?

File: stacksmith/user-scripts/boot.sh [hunk 3]
```diff
@@ -33,18 +37,24 @@ installWordPress() {
...
-        echo "==> WordPress is already installed"
-    else
-        configureWordPress
-        waitForDatabase
-        installWordPress
-    fi
+    prepareDataToPersist
+    configureWordPress
+    waitForDatabase
+    installWordPress
```
[the remark this answers is not in the document]
Comment (andresbono) [reply stacksmith/user-scripts/boot.sh]: wp-cli already checks that

File: stacksmith/user-scripts/build.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (marcosbc) [reply stacksmith/user-scripts/build.sh]: Only if UPLOADS_DIR is not persisted, which seems not to be the case.

File: stacksmith/user-scripts/boot.sh (hunk 1 above)
Comment (andresbono) [reply stacksmith/user-scripts/boot.sh]: Documented.

**Comment by andresbono** [comment]
Thanks for the review @absoludity. I'm going to update this PR to reflect the change that you are proposing for the built-in SM templates. Once that change is implemented, we can simplify the README of this example application. Let me know if you want anything else changed for this PR!

**Pull request merged**